packages/design-system/src/styles/components/_Button.scss

```diff
@@ -182,21 +182,38 @@
 %transparent-inverse-button {
   border-color: transparent;
   background-color: transparent;
+
+  &:disabled,
```

Comment:
I think disabled styles should be last, going after `active`

Reply:
Moved them last